lib/task/propel/propelBuildNestedSetTask.class.php

```diff
@@ -119,8 +123,8 @@ protected function configure()
         $this->briefDescription = 'Build all nested set values.';
 
         $this->detailedDescription = <<<'EOF'
-Build nested set values. Optionally excluding tables (information_object, term, menu).
-EOF;
+        Build nested set values. Optionally excluding tables (information_object, term, menu).
```

Comment:
I think these lines weren't indented because any extra spaces would show up on the command line when this is printed.